docs/tutorial/progress-bar.md

```diff
@@ -1,35 +1,69 @@
 # Progress Bar in Taskbar (Windows, macOS, Unity)
 
-On Windows a taskbar button can be used to display a progress bar. This enables
-a window to provide progress information to the user without the user having to
-switch to the window itself.
+## Overview
 
-On macOS the progress bar will be displayed as a part of the dock icon.
+A progress bar enables a window to provide progress information to the user
+without the need of switching to the window itself.
 
-The Unity DE also has a similar feature that allows you to specify the progress
-bar in the launcher.
+On Windows, you can use a taskbar button to display a progress bar.
 
-__Progress bar in taskbar button:__
+![Windows Progress Bar][windows-progress-bar]
 
-![Taskbar Progress Bar][taskbar-progress-image]
+On macOS, the progress bar will be displayed as a part of the dock icon.
 
-All three cases are covered by the same API - the `setProgressBar()` method
-available on instances of `BrowserWindows`. Call it with a number between `0`
-and `1` to indicate your progress. If you have a long-running task that's
-currently at 63% towards completion, you'd call it with `setProgressBar(0.63)`.
+![macOS Progress Bar][macos-progress-bar]
 
-Generally speaking, setting the parameter to a value below zero (like `-1`)
-will remove the progress bar while setting it to a value higher than one
-(like `2`) will switch the progress bar to intermediate mode.
+On Linux, the Unity graphical interface also has a similar feature that allows
+you to specify the progress bar in the launcher.
+
+![Linux Progress Bar][linux-progress-bar]
+
+> NOTE: on Windows, each window can have its own progress bar, whereas on macOS
+and Linux (Unity) there can be only one progress bar for the application.
+
+----
+
+All three cases are covered by the same API - the
+[`setProgressBar()`][setprogressbar] method available on an instance of
+`BrowserWindow`. To indicate your progress, call this method with a number
+between `0` and `1`. For example, if you have a long-running task that is
+currently at 63% towards completion, you would call it as
+`setProgressBar(0.63)`.
+
+Setting the parameter to negative values (e.g. `-1`) will remove the progress
+bar, whereas setting it to values greater than `1` (e.g. `2`) will switch the
+progress bar to indeterminate mode. In this mode, a progress bar remains active
+but does not show an actual percentage. Use this mode for situations when you
+do not know how long an operation will take to complete. The indeterminate mode
+is a Windows-only feature.
```

Comment:
```suggestion
Setting the parameter to negative values (e.g. `-1`) will remove the progress
bar, whereas setting it to values greater than `1` (e.g. `2`) will switch the
progress bar to indeterminate mode (Windows-only -- it will clamp to 100% otherwise).
In this mode, a progress bar remains active but does not show an actual percentage.
Use this mode for situations when you do not know how long an operation
will take to complete.
```